src/components/LocationSearch/LocationSearch.js

```diff
@@ -1,12 +1,13 @@
-import React, { useEffect, useState, useMemo } from "react";
-import { convertLocationObjectToArray } from "../../utils/dataTransformations";
+import React, { useEffect, useState, useMemo, useContext } from "react";
 import "./LocationSearch.css";
 import { APIUrl } from "../../config/config";
 import debounce from "lodash.debounce";
 import ResetButton from "../ResetButton/ResetButton";
+import CityList from "../CityList/CityList";
+import { AgencyContext } from "../../context/AgencyContext";
 
 export default function LocationSearch({ updateQuery, setSearch, city }) {
-  const [locations, setLocations] = useState({});
+  const { cityList, setCityList } = useContext(AgencyContext);
```

Comment:
This is an example of the context that I like: used in multiple components in a predictable and similar way 👍